editor/project_manager.cpp

```diff
@@ -317,7 +318,7 @@ class ProjectDialog : public ConfirmationDialog {
 
 		String p = p_path;
 		if (mode == MODE_IMPORT) {
-			if (p.ends_with("project.godot")) {
+			if (p.ends_with("projnpdialog->ect.godot")) {
```

Comment:
Copy paste mistake.

Reply:
Ping.

Reply:
I'm not quite understanding what's going on here. I'm new in this area of solving issues in open-source projects. Could you elaborate what you did and why you corrected that line of code?

Reply:
Thank you for the warning, I solved the issue!